msticpy/data/drivers/mdatp_driver.py

```diff
@@ -26,18 +27,29 @@ class MDATPDriver(OData):
     CONFIG_NAME = "MicrosoftDefender"
     _ALT_CONFIG_NAMES = ["MDATPApp"]
 
-    def __init__(self, connection_str: str = None, **kwargs):
+    def __init__(self, connection_str: str = None, instance: str = "Default", **kwargs):
         """
         Instantiate MSDefenderDriver and optionally connect.
 
         Parameters
         ----------
         connection_str : str, optional
             Connection string
+        instance : str, optional
+            The instance name from config to use
 
         """
         super().__init__(**kwargs)
-        api_uri, oauth_uri, api_suffix = _select_api_uris(self.data_environment)
+        cs_dict = _get_driver_settings(
+            self.CONFIG_NAME, self._ALT_CONFIG_NAMES, instance
+        )
+        self.cloud = cs_dict.pop("cloud", "global")
```

Comment:
Would be good to add this to the mpconfig yaml resource file so it shows up in the UI